docs/_snippets/features/horizontal-rule.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* globals window, document, console */
+
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic/src/ckeditor';
+import HorizontalRule from '@ckeditor/ckeditor5-horizontal-rule/src/horizontalrule';
+import { CS_CONFIG } from '@ckeditor/ckeditor5-cloud-services/tests/_utils/cloud-services-config';
+
+ClassicEditor.builtinPlugins.push( HorizontalRule );
+
+ClassicEditor
+	.create( document.querySelector( '#demo-editor' ), {
+		toolbar: {
+			items: [
+				'heading',
+				'|',
+				'bold',
+				'italic',
+				'bulletedList',
+				'numberedList',
+				'blockQuote',
+				'link',
+				'|',
+				'imageUpload',
+				'mediaEmbed',
+				'insertTable',
+				'|',
+				'undo',
+				'redo',
+				'|',
+				'horizontalRule'
```

Comment:
Having it after undo/redo is rather weird. I think it should end up somewhere higher. Probably after `insertTable`. We can then consider whether it should be surrounded by separators or adjacent to `insertTable.`.